### DISCUSSION TITLE
Error: Could not create PCH - ???

### DISCUSSION THREAD

**Discussion by itsamedood** [HaxeFlixel/flixel#2437] [Q&A] [answered]
- **Haxe version:** 4.2.3
- **Flixel version:** 4.9.0
- **OpenFL version:** Not sure how to check that-
- **Lime version:** Not sure how to check that either-
- **Affected targets:** MacOS
________________________________

I wrote a recreation of The World's Hardest Game (from CoolMath) for fun, and everything works fine, but my laptop (Macbook Air) needed to update, so I installed a software update, and now nothing works. I restarted VSC a thousand times, shutdown and restarted my laptop, but all I get when trying to run, test, or build is this error I've never seen or heard of before: **`Error: Could not create PCH`**. This happens to every game I've made, not just the recreation game mentioned above. Any ideas on how to fix this?
________________________________

**Comment by itsamedood** [comment, marked as answer]
Ok so after re-installing Xcode on my Macbook (the update got rid of it for whatever reason) it magically works now???

Yeah I'm confused but hey, all is good. 👌

**Reply by isakube** [reply]
xcode be like
